spec/request/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+describe 'Create Targets', type: :request do
+  let!(:user) { create(:user, :confirmed) }
+  let!(:auth_header) { user.create_new_auth_token }
+  let(:target) { build(:target) }
+  let(:params) do
```

Comment:
Checkout [attributes_for](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#using-factories).
It will allow you to do sth like: `let(:params) { attributes_for(:target) }`